### DISCUSSION TITLE
Scroll bar is barely visible

### DISCUSSION THREAD

**Discussion by gtirloni** [community/community#11360] [Other Feature Feedback, Questions, & Ideas] [answered]
Unless the mouse pointer is over it, you can barely see the vertical scroll bar.

![image](https://user-images.githubusercontent.com/5902830/153575512-7636f34f-7a62-4e95-b805-e1fd0c3a037f.png)

**Comment by colinwm** [comment, marked as answer]
Good point, it is a bit hard to see. I've made the contrast a bit stronger.
